src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -36,6 +37,8 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
     private maxNonStatusRetry: number = 60; // retries for max 5 min
     private localGitRepoHelper: LocalGitRepoHelper;
     private filesToCommit: DraftFile[] = [];
+    private committedWorkflow: string;
+    private tmpDirectoryPath: string;
```

Comment:
give a better name of what this temp is for. for example `tempWorkflowDirPath`

Reply:
done